### DISCUSSION TITLE
is there any good solution to display an pdf?

### DISCUSSION THREAD

**Discussion by Nebdir** [emilk/egui#2206] [Q&A]
I thought it would be nice to show a generated pdf inside the app. A little research didn't offer a direct solution, and neither did research on the issues and discussions. So I thought it would be pleasant for anyone searching for a similar case to find at least this question here. 

But back to the topic, did I miss any already existing solution? Or where should I start to add an element like this?

**Comment by LEXUGE** [comment]
I am also very interested in this

**Comment by luishugoooo** [comment]
same

**Comment by hrl10** [comment]
If you just want to preview the pdf file,you can use pdfium-render to convert the pdf file to DynamicImage and render it

**Reply by White-Rabbit-Scientific** [reply]
https://github.com/ajrcarey/pdfium-render
